libyara/modules/dotnet.c

```diff
@@ -1661,6 +1685,9 @@ begin_declarations;
   declare_string_array("constants");
   declare_integer("number_of_constants");
 
+  declare_integer_array("field_offset");
```

Comment:
Array names are usual in plural form, this should be named `field_offsets` for consistency. Also this should be added to the documentation.